### PR TITLE
KING-1.4_Linux-x86_64.eb (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/60fa273716f9242c7fab for a full test report.

File: easybuild/easyconfigs/k/KING/KING-1.4_Linux-x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+sources = ['Linux-%(namelower)s.tar.gz']
+
+sanity_check_paths = {
+    'files': ["king"],
+    'dirs': []
+}
+
+# add the installation dir to PATH
+modextrapaths = { 
+    'PATH': "",
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/k/KING/KING-1.4_Linux-x86_64.eb]: pr is nearly ready; check above line if you wish make it dict-like

**Comment by fgeorgatos** [comment]
put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**